generic/BRGenericRipple.c

```diff
@@ -73,8 +71,6 @@ genericRippleAccountSignTransferWithKey (BRGenericAccountRef account,
     // Get the transaction pointer from this transfer
     BRRippleTransaction transaction = rippleTransferGetTransaction ((BRRippleTransfer) transfer);
     if (transaction) {
-        // Hard code the sequence to 7
-        rippleAccountSetSequence ((BRRippleAccount) account, 7);
 //        rippleAccountSignTransaction(account, transaction, seed);
         assert (0);
```

Comment:
Should we just remove the function instead?

Reply:
We could.